.github/workflows/test-playwright.yml

```
@@ -0,0 +1,153 @@
+              echo "Waiting for CANChat..."
+              sleep 5
+            else
+              echo "CANChat is ready."
+              exit 0
+            fi
+ 
+          done
+          echo "CANChat failed to start in time" >&2
+          exit 1
+ 
+      - name:  Start Playwright Tests
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)